microprofile.ls/CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Quarkus Language Server Changelog
```

Comment:
@fbricon change to "MicroProfile Language Server Changelog"?

Reply:
yes